test/integ/conftest.py

```diff
@@ -12,6 +12,20 @@
 
 _BLENDER_VERSION_RE = re.compile(r"^Blender (?P<version>\d+\.\d+)\..*$")
 
+# Set BLENDER_EXECUTABLE based on platform and version
+blender_version_env = os.environ.get("BLENDER_VERSION")
+if blender_version_env and not os.environ.get("BLENDER_EXECUTABLE"):
+    if os.name == "nt":
+        os.environ["BLENDER_EXECUTABLE"] = (
+            f"C:\\Tools\\blender-{blender_version_env}-windows-x64\\blender.exe"
+        )
+    elif sys.platform == "darwin":
+        os.environ["BLENDER_EXECUTABLE"] = (
+            f"/Applications/Blender-{blender_version_env}.app/Contents/MacOS/Blender"
+        )
+    else:  # Linux
+        os.environ["BLENDER_EXECUTABLE"] = f"/opt/blender-{blender_version_env}-linux-x64/blender"
```

Comment:
This code should probably be moved into the `blender_location` fixture rather than manipulating the `BLENDER_EXECUTABLE` env var